lib/mix/tasks/compile.make.ex

```diff
@@ -83,6 +83,11 @@ defmodule Mix.Tasks.Compile.ElixirMake do
   and `mix test` commands.
   """
 
+  @exec_notfound_msg """
```

Comment:
All other error messages in the project are inlined here. So isn't this correct?